geode-web-management/src/main/java/org/apache/geode/management/internal/rest/security/RestSecurityConfiguration.java

```diff
@@ -66,6 +69,22 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
     return super.authenticationManagerBean();
   }
 
+  @Bean
+  public MultipartResolver multipartResolver() {
```

Comment:
do we have any test coverage for this new method?

Reply:
yes, we have integration test and dunit test that test this.